doc/introduction/operations.rst

```diff
@@ -25,12 +25,37 @@ These operations can be used exclusively in quantum functions, like shown in the
 
 This quantum function uses the :class:`RZ <pennylane.RZ>`,
 :class:`CNOT <pennylane.CNOT>`,
-:class:`RY <pennylane.RY>` :ref:`gates <intro_ref_ops_qgates>`, the
+:class:`RY <pennylane.RY>` gates, the
 :class:`AmplitudeDamping <pennylane.AmplitudeDamping>`
-:ref:`noisy channel <intro_ref_ops_channels>` as well as the
-:class:`PauliZ <pennylane.PauliZ>` :ref:`observable <intro_ref_ops_qobs>`.
+noisy channel as well as the
+:class:`PauliZ <pennylane.PauliZ>` observable.
 
-Below is a list of all quantum operations and operation functions supported by PennyLane.
+Functions applied to operators extract information (such as the matrix representation) or
+transform operators (like turning a gate into a controlled gate).
+
+PennyLane supports the following operators and operator functions:
+
+:ref:`intro_ref_ops_funcs`
+
+:ref:`intro_ref_ops_qubit`:
+
+* :ref:`intro_ref_ops_nonparam`
+* :ref:`intro_ref_ops_qparam`
+* :ref:`intro_ref_ops_qchem`
+* :ref:`intro_ref_ops_matrix`
+* :ref:`intro_ref_ops_arithm`
+* :ref:`intro_ref_ops_qstateprep`
+* :ref:`intro_ref_ops_channels`
+* :ref:`intro_ref_ops_qobs`
+
+:ref:`intro_ref_ops_cv`:
+
+* :ref:`intro_ref_ops_cvgates`
+* :ref:`intro_ref_ops_cvstateprep`
+* :ref:`intro_ref_ops_cvobs`
```

Comment:
@mariaschuld I wonder if this is necessary (and may be harder to maintain), since there is already a local TOC that is visible at all times on the right hand side:

![image](https://user-images.githubusercontent.com/2959003/168189368-48c54ec7-dd7c-48d9-9495-4e1ccc8837f3.png)

Reply:
Oh, I totally didn't see that. Will delete the manual version!